packages/connect-web/src/grpc-web-transport.ts

```diff
@@ -219,6 +219,8 @@ export function createGrpcWebTransport(
           }
           return <UnaryResponse<I, O>>{
```

Comment:
I think we can remove the cast here
```suggestion
          return {
```

Reply:
👍 thx for review